Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.35442&attempt_id=728662dc-cd0a-4887-a265-c3ef84b9cee9)

<details><summary>Partial log (click to expand)</summary><p>

```
 PASS   reindex detects removal of all files
 PASS   reindex preserves properties

Notmuch test suite complete.
955/960 tests passed.
4 broken tests failed as expected.
1 test failed.
make: *** [test/Makefile.local:71: test] Error 1
builder for '/nix/store/bcai63zwbz18rcvcca56kslrkyq6zcp6-notmuch-0.26.drv' failed with exit code 2
[31;1merror:[0m build of '/nix/store/bcai63zwbz18rcvcca56kslrkyq6zcp6-notmuch-0.26.drv' failed
```
</p></details>

--- a/pkgs/applications/networking/mailreaders/notmuch/default.nix
+++ b/pkgs/applications/networking/mailreaders/notmuch/default.nix
@@ -54,6 +54,10 @@ stdenv.mkDerivation rec {
     done
   '';
 
+  # Notmuch doesn't use autoconf and consequently doesn't tag --bindir and
+  # friends
+  setOutputFlags = false;
+  enableParallelBuilding = true;
   makeFlags = "V=1";
 
   preFixup = optionalString stdenv.isDarwin ''

